test/e2e/networks/single.toml

```diff
@@ -1 +1,5 @@
-[node.validator]
+vote_extensions_update_height = -1
+vote_extensions_enable_height = 1
+
```

Comment:
Do we want to keep this?

Reply:
leftover from local test runs. will revert